src/store/CurrentAccount/actions.ts

```diff
@@ -0,0 +1,41 @@
+import {CurrentAccountInterface} from "@/interfaces";
```

Comment:
Lint will ask for spaces after { and before }.

Reply:
Also I think we don't need the semi-colons.

Reply:
Good catch, I'll fix it.